tools/vcf_to_aliquot_maf.cwl

```diff
@@ -1,18 +1,20 @@
-#!/usr/bin/env cwl-runner
 
 cwlVersion: v1.0
-
 class: CommandLineTool
 requirements:
   - class: DockerRequirement
-    dockerPull: quay.io/ncigdc/aliquot-maf-tools:f7a5a7d87abd12cc6b644698bb90854bc9679a01
+    dockerPull: quay.io/ncigdc/aliquot-maf-tools:1.0.1-rc.8-3-g9e5d940
```

Comment:
Different tag than the merge https://github.com/NCI-GDC/aliquot-maf-cwl/pull/7/files#diff-26c9196f118a6229d198363d2458cad3335e268b3af82a5f19097a691812d950R8

Reply:
fixed